docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
 # Getting started
 
-TODO
+In this tutorial, we will download and install Pebble, configure layers, run the Pebble daemon, and work with layers and services. After this tutorial, you will have a basic understanding of what Pebble is, how to install it, and how to use it to orchestrate services. It takes about 15 minutes to complete.
+
+
+After that, you can continue exploring more advanced features and use cases (links at the end).
+
+## Prerequisites
+
+- A Linux machine.
+
+## Download and install Pebble
+
+Find the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following commands to download, extract, and install the latest release (replace `v1.11.0` with the latest tag and `amd64` with your architecture):
+
+```bash
+wget https://github.com/canonical/pebble/releases/download/v1.11.0/pebble_v1.11.0_linux_amd64.tar.gz
+tar zxvf pebble_v1.11.0_linux_amd64.tar.gz
+sudo mv pebble /usr/local/bin/ # make sure it's in your $PATH
+```
+
+## Verify Pebble installation
+
+Once the installation is complete, verify that `pebble` has been installed correctly by running:
+
+```bash
+pebble
+```
+
+This should produce output similar to the following:
+
+```bash
+Pebble lets you control services and perform management actions on
+the system that is running them.
+
+Usage: pebble <command> [<options>...]
+
+...
+```
+
+## Configure Pebble
+
+Now that Pebble has been installed, we can set up a basic configuration.
+
+First, let's create a directory for Pebble configuration:
+
+```bash
+mkdir -p ~/PEBBLE/layers
+export PEBBLE=$HOME/PEBBLE
+echo "export PEBBLE=$HOME/PEBBLE" >> ~/.bashrc # add $PEBBLE to your bashrc
+```
+
+Next, create a layer by running:
+
+```bash
+echo """\
+summary: Simple layer
+
+description: |
+    A simple layer.
+
+services:
+    http-server:
+        override: replace
+        summary: demo http server
+        command: python3 -m http.server 8080
+        startup: enabled
+""" > $PEBBLE/layers/001-http-server.yaml
+```
+
+This creates a simple layer containing only one service (which runs a basic HTTP server using Python's `http` module, listening on port 8080).
+
+## Start the Pebble daemon
+
+Now we are ready to run the Pebble daemon.
+
+Pebble is invoked using `pebble <command>`. (To get more information, run `pebble -h`.)
```

Comment:
```suggestion
```{note}
Pebble is invoked using `pebble <command>`. To get more information, run `pebble -h`.
```
```

I still think this is conflating explanation when it is not needed at this point of the tutorial. But if you want to keep this here I suggest putting it inside an alert box since it is an additional piece of information that the reader can choose to read or skip.

And this note box should be moved ahead before the "Now we are ready to run ..." sentence.

Reply:
It is necessary, because most command goes like `command <args>`, while pebble, like kubectl, goes like `command subcommand <args>`. It's worth mentioning that pebble has sub commands before running it for the first time.